src/pages/project-grid.js

```diff
@@ -11,13 +12,41 @@ import Footer from "../sections/Footer";
 import { GlobalStyle } from "../sections/app.style";
 import theme from "../theme/blog/themeStyles";
 
-const ProjectGridPage = () => (
+export const query = graphql`
+    query allProjects {
+        allMdx(
+            sort: { fields: [frontmatter___date], order: DESC }
+            filter: { fields: { collection: { eq: "projects" } }, frontmatter: { published: { eq: true } } }
+        ) {
+            nodes {
+                id
+                frontmatter {
+                    subtitle
```

Comment:
Why not use title?

Reply:
Because I thought titles would be too large to display and we are anyways displaying them on single project page. Any feedback using shorter titles @leecalcote @Nikhil-Ladha @Tanuj22